Xamarin.Forms.Platform.UAP/VisualElementPackager.cs

```diff
@@ -118,7 +116,16 @@ void OnChildAdded(object sender, ElementEventArgs e)
 				Windows.UI.Xaml.Controls.Grid.SetColumnSpan(childRenderer.ContainerElement, _columnSpan);
 
 			_panel.Children.Add(childRenderer.ContainerElement);
+		}
+
+		void OnChildAdded(object sender, ElementEventArgs e)
+		{
+			var view = e.Element as VisualElement;
+
+			if (view == null)
+				return;
 
+			SetupVisualElement(view);
 			if (ElementController.LogicalChildren[ElementController.LogicalChildren.Count - 1] != view)
```

Comment:
Suggested fix: 

In EnsureZIndex(), set a private variable _hasZIndexes to true. Then change this line to:

`if (_hasZIndexes || ElementController.LogicalChildren[ElementController.LogicalChildren.Count - 1] != view)`